modules/avatar/hash_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 The Forgejo Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2023 The Gitea Authors. All rights reserved.
```
I think the CI won't pass otherwise.

Reply:
As a point of clarification, will you be able to provide the copyright assignment? By introducing code from another project without the copyright assignment it raises many questions including "are we required to add the notice to the license file in the root of the git directory", among others.

Reply:
I propose to just do it as with other gogs code ...
... two lines

Reply:
I do not think it makes sense to add more and more headers.

Otherwise I would add my personal name to this header like "Copyright the Gitea Authors, Copyright wxiaoguang"

Reply:
Well we are talking about projekt's and not about individual contributors ... but i get your concern